### PR TITLE
fill out start game interface and add tests

### LINKED ISSUE #61: Start game interface

### PR DESCRIPTION
**Pull request by hrice9**
when a user is an admin in an org with no game, show a start game button and a modal to create a new game.

Fixes #61

### CODE REVIEW

File: HVZ.Tests/HVZ.Web.Components.Tests/StartGameTest.razor [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        input.Change(testGameName);
+
+        var btn = cut.Find("#btn-new-game");
+        btn.Click();
+        
+        // TODO: Figure out how Verify actually works
+        // Currently saying that this is not being called at all
+        //mockOrgRepo.Verify(
+        //    r => r.CreateGame(testGameName, testOrgId, testUserId)
+        //);
```
Comment (Mogiiii) [inline HVZ.Tests/HVZ.Web.Components.Tests/StartGameTest.razor:100]: ?

Comment (hrice9) [reply HVZ.Tests/HVZ.Web.Components.Tests/StartGameTest.razor:100]: `mockOrgRepo.Verify(...)` is not behaving as expected. The following test checking for the error message is good enough for now, I would like to check that the database method is actually being called however

Comment (Mogiiii) [reply HVZ.Tests/HVZ.Web.Components.Tests/StartGameTest.razor:100]: Is this required for the test to work or can it be safely remoced and investigated later?

if so, lets do that and we can merge this pr

Comment (hrice9) [reply HVZ.Tests/HVZ.Web.Components.Tests/StartGameTest.razor:100]: it is not required and can be removed

**Pull request merged**